src/utils/timeDetail.ts

```diff
@@ -0,0 +1,17 @@
+import moment from 'moment';
+
+export function getTimeDetail(endTime: moment.Moment) {
+  const ONE_MINUTE = 60000;
+
+  const currentTime = moment();
+  let differenceInMilliseconds = currentTime.diff(endTime);
+  // manual fix because moment.js is returning one second instead of 'a few seconds'
+  let timeDetailHumanized =
+    Math.abs(differenceInMilliseconds) >= ONE_MINUTE
+      ? moment.duration(differenceInMilliseconds).humanize()
+      : 'a few seconds';

```

Comment:
We need a translation here.

To use translation outside hooks, use:
`i18next.t('translationKey')`

Reply:
woow i missed this one, thanks!